ui/admin/tests/acceptance/users/create-test.js

```diff
@@ -84,31 +84,33 @@ module('Acceptance | users | create', function (hooks) {
       );
     await visit(urls.orgScope);
 
-    await click(`[href="${urls.users}"]`);
+    await click(commonSelectors.HREF(urls.users));
 
     assert.false(
       instances.scopes.org.authorized_collection_actions.users.includes(
         'create',
       ),
     );
 
-    assert.dom('.rose-button-primary').doesNotExist();
+    assert.dom(commonSelectors.HREF(urls.newUser)).doesNotExist();
```

Comment:
Nice catch. I don't remember what `.rose-button-primay` is but now I know exactly what this test is asserting.